.github/workflows/linux.yml

```diff
@@ -143,6 +153,8 @@ jobs:
             -DCMAKE_Fortran_COMPILER=$(which gfortran-10)
         make -j 2
 
+        ctest --output-on-failure -E Particles_GhostsAndVirtuals
```

Comment:
Those should become addressed with #2247 (issue: #2243).
We can comment those in again once the linked PR is merged